cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -43,23 +42,14 @@ var createClusterCmd = &cobra.Command{
 
 func init() {
 	createCmd.AddCommand(createClusterCmd)
-	createClusterCmd.Flags().StringVarP(&cc.fileName, "filename", "f", "", "Filename that contains EKS-A cluster configuration")
-	createClusterCmd.Flags().StringVarP(
-		&cc.hardwareCSVPath,
-		TinkerbellHardwareCSVFlagName,
-		TinkerbellHardwareCSVFlagAlias,
-		"",
-		TinkerbellHardwareCSVFlagDescription,
-	)
+	setupClusterOptionFlags(createClusterCmd, &cc.clusterOptions)
+	setupTimeoutFlags(createClusterCmd, &cc.timeoutOptions)
+	addTinkerbellFlag(createClusterCmd, &cc.hardwareCSVPath)
```

Comment:
I don't have a strong preference on the verb, but I would suggest using the same one in all methods